dts/arm/atmel/sam3u.dtsi

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <freq.h>
+#include <arm/armv7-m.dtsi>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+
+/ {
+	chosen {
+		zephyr,flash-controller = &efc0;
+	};
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m3";
+			reg = <0>;
+			clock-frequency = <DT_FREQ_M(96)>;
+		};
+	};
+
+	soc {
+		usart0: usart@40090000 {
+			compatible = "atmel,sam-usart";
+			reg = <0x40090000 0x130>;
+			interrupts = <13 0>;
+			peripheral-id = <13>;
```

Comment:
Hi @xingrz ,

Please, rebase your changes on top of main and update your PR to use clocks instead peripheral-id.